x11-wm/i3-gaps/i3-gaps-4.19.1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson optfeature
+
+DESCRIPTION="i3 fork with gaps and some more features"
+HOMEPAGE="https://github.com/Airblader/i3"
+SRC_URI="https://github.com/Airblader/i3/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="dev-libs/glib:2
+	dev-libs/libev
+	dev-libs/libpcre
+	dev-libs/yajl
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/libxcb[xkb]
+	x11-libs/libxkbcommon[X]
+	x11-libs/pango[X]
+	x11-libs/startup-notification
+	x11-libs/xcb-util
+	x11-libs/xcb-util-cursor
+	x11-libs/xcb-util-keysyms
+	x11-libs/xcb-util-wm
+	x11-libs/xcb-util-xrm"
+DEPEND="${COMMON_DEPEND}
+	test? (
+		dev-perl/ExtUtils-PkgConfig
+		dev-perl/IPC-Run
+		dev-perl/Inline
+		dev-perl/Inline-C
+		dev-perl/X11-XCB
+		dev-perl/XS-Object-Magic
+		x11-apps/xhost
+		x11-base/xorg-server[xephyr,xvfb]
+		x11-misc/xvfb-run
+	)"
+BDEPEND="app-text/asciidoc
```

Comment:
Oh yea I wanted to ask if asciidoc is required everytime, or just when `USE="doc"` is wanted?

Reply:
Sorry it took so long to respond! `i3-gaps` doesn't have USE=doc, but `i3` itself does. I don't see much of a need for it in any case, given `qsize` reports `x11-wm/i3: 64 files, 15 non-files, 2243 KiB` with USE=-doc and `x11-wm/i3: 66 files, 16 non-files, 2272 KiB` with USE=doc. A difference of a couple dozen kilobytes doesn't necessitate adding the flag, I think.